### DISCUSSION TITLE
Local development requirements packages

### DISCUSSION THREAD

**Discussion by moas** [cookiecutter/cookiecutter-django#5209] [Q&A] [answered]
why did you decide to install the packages needed for production in local development environment ?

**Comment by foarsitter** [comment, marked as answer]
Because you develop for production. 

I'm using the `capture_exception` function of sentry a lot. It should mean that I cannot run my code locally without installing the `sentry-sdk`. Of course I can work around by writing code but I want my environments as equal as possible.

Another thing that comes to my mind is that a production dependency can be incompatible with you local stack. That is not something I want to find out in production. I know, it is highly theoretical, but installing the dependencies has no downside to me. 

Feel free to read some about the change over here: https://github.com/cookiecutter/cookiecutter-django/pull/4838
